core/ui/src/views/SoftwareCenterAppInstances.vue

```diff
@@ -323,6 +323,17 @@
                       })
                     }}
                   </div>
+                  <template v-if="app.docs.relnotes_url">
```

Comment:
Since we display the link on a new line, I'd change this `<template>` to a `<div>` and remove the `<span>` element.

Does this release notes link always appear? Shouldn't it appear only if an update is available?